target/maven-archiver/pom.properties

```diff
@@ -1,5 +1,5 @@
 #Generated by Maven
-#Mon Jun 22 22:22:45 UTC 2020
+#Thu Jun 25 17:13:51 UTC 2020
```

Comment:
What is this file for again? I havent used maven in a long time.
If this is something generated during builds and not something you actually are updating. I'd add it to a git ignore file to your repo, so git ignores it.